backend/.env.sample

```diff
@@ -3,3 +3,4 @@ DB_HOST=localhost
 DB_USER=notroot
 DB_PASSWORD=helloworld
 DB_NAME=mvc_express
+TOKEN_SECRET=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiO…Tk0fQ.4FjW
```

Comment:
Don't push online secret and db server credentials...

Reply:
It's a completely random password hash used as an exemple, and the DB_NAME is the default name found in env.sample.